src/pages/ReimbursementAccount/ReimbursementAccountForm.js

```diff
@@ -58,9 +58,9 @@ class ReimbursementAccountForm extends React.Component {
                     onFixTheErrorsLinkPressed={() => {
                         this.form.scrollTo({y: 0, animated: true});
                     }}
-                    message={this.props.reimbursementAccount.error}
+                    message={this.props.reimbursementAccount.error || _.values(this.props.reimbursementAccount.errors)[0]}
```

Comment:
Why are we accessing errors both ways now? Is this a temporary thing while a Web-E PR updating error handling goes out?

Reply:
Yes, this is a temporary thing since other steps in the VBA flow use `this.props.reimbursementAccount.error`